app/assets/javascripts/angular/filters/hyphens.coffee

```diff
@@ -0,0 +1,4 @@
+angular.module "lean-coffee"
+  .filter 'hyphens', ($sce) ->
+    (str) ->
+      str.replace(/\s+/g, '-')
```

Comment:
It's the lil' red icon that GitHub adds that indicates a missing newline. Would be nice to fix :smile: